app/design/frontend/base/default/layout/boltpay.xml

```diff
@@ -29,22 +29,26 @@
 
   <default>
     <reference name="head">
+      <action method="addItem" ifconfig="payment/boltpay/add_button_to_minicart"><type>skin_css</type><name>css/boltpay/minicart.css</name><params/></action>
       <block type="boltpay/checkout_boltpay" name="track_script" as="track_script" template="boltpay/track.phtml" />
+      <block type="boltpay/checkout_boltpay" name="connect_js_script" as="connect_js" template="boltpay/connectjs.phtml" />
```

Comment:
Since we have `add_button_to_minicart` config, can we make sure this only included when `add_button_to_minicart` is true or in cart page?